### PR TITLE
Services

### PR DESCRIPTION
**Pull request by manzitresor**
# Service Page

In this Project we have implemented the following features:

## Description of tasks that were expected to be completed

List all the tasks and sub-tasks that were assigned to you Make sure you check every completed task. list them in this style

- [x] Create HTML structure of Service page
- [x] Add Styles
- [x] Add mobile
- [x] Add iPad
- [x] Add desktop

## Github board link
[https://github.com/manzitresor/team4-website-project-real](url)

Provide github board link related to your PR in case it's appropriate and applicable

## Please check this Checklist before you submit your PR:

- [x]  My code follows the style guidelines of this project
- [x]  I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My code generates no warnings
- [x] There are no vulnerabilities

### CODE REVIEW

**Comment by iAmNsengi** [comment]
A small recommendation:
The font-semibold applied to every h5 would be more readable if it were set on the parent div, but I think it is still going to work in the same way.

File: src/sections/services.html [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>services</title>
-</head>
-<body class="font-outfit">
-    <h1 class="text-blue-500">Services</h1>
-</body>
-</html>
\ No newline at end of file
+<div class="grid my-5 grid-cols-1 gap-20 px-8 md:px-20 font-outfit lg:grid-cols-2">
```
Comment (kareraolivier) [inline src/sections/services.html:1]: - [ ] This page need to have the page height

File: src/sections/services.html [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
+    <div class="flex flex-col gap-y-5 text-2xl md:text-4xl lg:gap-y-14">
+        <h5 class="font-semibold ">Brand strategy</h5>
+        <h5 class="font-semibold ">Corporate Identity & <br class="md:hidden"> Website development</h5>
+        <h5 class="font-semibold ">SEO & copywriting</h5>
+        <h5 class="font-semibold ">Media Relations</h5>
+        <h5 class="font-semibold ">Digital marketing &<br class="md:hidden">content production</h5>
+        <h5 class="font-semibold ">Influencer relations</h5>
```
Comment (kareraolivier) [inline src/sections/services.html:8]: - [ ]  Are they the titles or the list? How do you see it as a group?

**Review by kareraolivier — CHANGES_REQUESTED** [review]
- [ ] The PR is not updated from the dev branch it is behind DEV
- [ ] Also work on responsiveness here:
Screenshot Capture - 2024-05-31 - 11-54-27

File: src/sections/services.html (hunk 2 above)
Comment (iAmNsengi) [reply src/sections/services.html:8]: On my side, it would be better if they were <li>s in a <ul>.

Comment (kareraolivier) [reply src/sections/services.html:8]: Yes, that is what I see, too. you can check in with a group to see if you can implement it.

**Comment by manzitresor** [comment]
> * [x]  The PR is not updated from the dev branch it is behind DEV
> * [ ]  Also work on responsiveness here:
>   Screenshot Capture - 2024-05-31 - 11-54-27

Can we know the screen which is not responsive

File: src/index.html [hunk 3]
```diff
@@ -5,27 +5,42 @@
...
+                    service();       
+                }
+            };
+
+            loadComponents('header', 'sections/header.html', componentLoaded);
+            loadComponents('hero', 'sections/hero.html', componentLoaded);
+            loadComponents('about', 'sections/about.html', componentLoaded);
+            loadComponents('sticky', 'sections/sticky-yellow.html', componentLoaded);
+            loadComponents('services', 'sections/services.html', componentLoaded);
+            loadComponents('references', 'sections/references.html', componentLoaded);
```
Comment (kareraolivier) [inline src/index.html:34]: All of this are on dev branch I should not see it in this branch 
the rest all ok

**Review by kareraolivier — CHANGES_REQUESTED** [review]
This branch is behind the dev branch 
did you merge dev?

**Review by kareraolivier — APPROVED** [review]
It looks good 
Conglatulation

**Pull request merged**